text/0000-core-builder.md

```diff
@@ -0,0 +1,46 @@
+# Ubuntu Core Builder
+
+## Summary
+
+Over the past, the Paketo project has created four different stacks and builders based on the Ubuntu Linux distribution. The difference is in the number of packages included which in turn is based on the requirements of the different buildpacks included.
+This RFC proposes to create a new stack repository that will create a single `build` image and a number of `run` images based on the Ubuntu distribution and replace the existing `static`, `tiny` and `base` stacks.
```

Comment:
I think it would be preferable to have run images based on the engine (e.g. nodejs, java, etc.) This would minimize to the absolute necessary, the packages that are needed for the engine during the runtime. We can also still have the static, tiny and the base for backwards compatibility.

Reply:
Indeed, I would consider the `static`, `tiny`, and `base` run images as kind of a means for compatibility. Although `base` also serves as an enabler for "works out of the box" (see my comment above).

Reply:
```suggestion
This RFC proposes to create a new stack repository that will create a single `build` image and a number of `run` images based on the Ubuntu distribution and replace the existing `static`, `tiny` and `base` stacks. The `full` stack will be deprecated immediately, making the Jammy stack the last iteration of it.
```